package.json

```diff
@@ -27,11 +27,13 @@
   },
   "dependencies": {
     "@airbattle/protocol": "github:wight-airmash/ab-protocol#v6.0.0",
+    "@types/netmask": "^1.0.30",
```

Comment:
this is the only thing i'm not sure about here. are types just a dev dependency?

Reply:
Depends on how you use it, in this case it is dev dependency, I will fix it.